bash_scripts/batch_atm.NERSC.bash

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -l
+#SBATCH --partition=regular
```

Comment:
I don't think the atmospheric diagnostics are going to work as expected when they run on 1 processor on 1 node on the `regular` partition.  I think they will end up running in serial.

To take advantage of the way they are coded (i.e. launching several tasks and then waiting for them to complete), you will probably need to use the `shared` partition:
http://www.nersc.gov/users/computational-systems/edison/running-jobs/example-batch-scripts/#toc-anchor-13
I am not exactly sure how this will need to work, though.  My sense is that you will need to request more than one processor and maybe a certain amount of memory, as shown in one of the examples.  Anyway, I think this needs some work and testing for sure and should not be rushed!